backend/kesaseteli/companies/api/v1/views.py

```diff
@@ -66,11 +66,12 @@ def get(self, request: Request, format: str = None) -> Response:
         business_id = organization_roles.get("identifier")
 
         try:
-            company = get_or_create_company_with_business_id(business_id)
-        except HTTPError:
-            return self.ytj_api_error
+            company = get_or_create_company_from_ytj_api(business_id)
         except ValueError:
             return self.ytj_response_error
+        except HTTPError:
+            name = organization_roles.get("name")
+            company = get_or_create_company_with_name_and_business_id(name, business_id)
```

Comment:
In case it is an transient error (YTJ API under too heavy load or something else), will the application logic re-try at some point (such as, when user performs some action in future?)

Reply:
At the moment it is not retried. If the information is not found from YTJ / an error occurs, the user has to fill in the information themselves. We still have to discuss what happens in the case of an error and if we want to have some kind of special handling for that.